elvee.c

```diff
@@ -455,3 +459,70 @@ void timestamp()
     month = (strstr(month_names, month_name) - month_names) / 4 + 1;
     printf("%04hu-%02hu-%02hu %02hu:%02hu:%02hu\n", year, month, day, hour, min, sec);
 }
+
+// Adapted from https://docs.microsoft.com/en-us/archive/blogs/twistylittlepassagesallalike/everyone-quotes-command-line-arguments-the-wrong-way
+// See issue #1
+// The returned pointer is a new allocation, but leaking it is fine.
+char *argv_quote(char *arg) {
```

Comment:
For consistency with the rest of the code format:

```suggestion
char *argv_quote(char *arg)
{
```